From f06d897d58b45cd8004af5a22a64d3fed52796b8 Mon Sep 17 00:00:00 2001
From: Lionel Xu <Lionel.Xu@freescale.com>
Date: Wed, 25 May 2011 12:11:36 +0800
Subject: [PATCH 155/635] ENGR00143658 GPS: Add gps support on gingerbread

Fix the compile error for other platform than imx53smd.

Signed-off-by: Lionel.Xu <R63889@freescale.com>
---
 libgps/Android.mk | 25 +++++++++++--------------
 1 file changed, 11 insertions(+), 14 deletions(-)

diff --git a/libgps/Android.mk b/libgps/Android.mk
index df10b02..0958535 100644
--- a/libgps/Android.mk
+++ b/libgps/Android.mk
@@ -1,19 +1,16 @@
 LOCAL_PATH := $(call my-dir)
-
 ifneq ($(TARGET_PRODUCT),sim)
 # HAL module implemenation, not prelinked and stored in
 # hw/<GPS_HARDWARE_MODULE_ID>.<ro.hardware>.so
-include $(CLEAR_VARS)
-LOCAL_PRELINK_MODULE := false
-LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
-LOCAL_SHARED_LIBRARIES := liblog libcutils libhardware
-
-ifeq ($(USE_ATHR_GPS_HARDWARE),true)
-    LOCAL_SRC_FILES := athr_gps.c
-endif
-
-LOCAL_SRC_FILES += gps.c
-LOCAL_MODULE := gps.default
-LOCAL_MODULE_TAGS := eng
-include $(BUILD_SHARED_LIBRARY)
+    ifeq ($(USE_ATHR_GPS_HARDWARE),true)
+        include $(CLEAR_VARS)
+        LOCAL_PRELINK_MODULE := false
+        LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
+        LOCAL_SHARED_LIBRARIES := liblog libcutils libhardware
+        LOCAL_SRC_FILES := athr_gps.c
+        LOCAL_SRC_FILES += gps.c
+        LOCAL_MODULE := gps.default
+        LOCAL_MODULE_TAGS := eng
+        include $(BUILD_SHARED_LIBRARY)
+    endif
 endif
-- 
1.8.0

